specs/service.spec

```
@@ -296,6 +296,12 @@ attributes:
     exposed: true
     stored: true
 
+  - name: proxyProtocolPreserveHeader
+    description: Do not strip proxy protocol header.
```

The name could be reworded to highlight that it is a boolean from the start. Maybe something like this?
```suggestion
  - name: preserveProxyProtocolHeader
    description: If true, the proxy protocol header will not be stripped.
```

---

I chose this name because the other proxy protocol fields also start with "proxyProtocol"...